django_girls.html

```diff
@@ -25,8 +25,7 @@
         <div class="reveal">
             <div class="slides">
                 <section>
-                    <h1>Code for Poznan </h1>
```

Comment:
I was rather thinking about moving logo here and removing secons slide.

Something like this:
```
                <section>
                    <section>
                        <h1>
                            <img src="img/logo_code_for_poznan.png" style="background:none; border:none; box-shadow:none; "/>
                            <h3>Django Girls</h3>
                            <h4>24.06.2017, Poznań</h4>
                        </h1>
                    </section>
                        <section>
                        <h2>Misja</h2>
                        <p class="fragment">Technologiczne rozwiązania społecznych problemów</p>
                        </section>
                </section>
```